scripts/zkai_sota_artifact_watchlist_gate.py

```
@@ -0,0 +1,894 @@
+            if "not a matched local benchmark" not in row["recommended_use"]:
+                raise SotaWatchlistError("NANOZK must remain non-matched context")
+        if system == "Obelyzk":
+            if row["status"] != STATUS_DEPLOYMENT_CONTEXT:
+                raise SotaWatchlistError("Obelyzk status drift")
+            if "not local verifier-time comparison" not in row["recommended_use"]:
+                raise SotaWatchlistError("Obelyzk local verifier-time overclaim")
+        if system in SETTLEMENT_OR_ZKVM_SYSTEMS and row["status"] == STATUS_EMPIRICAL:
+            raise SotaWatchlistError(f"settlement/zkVM context promoted to empirical adapter: {system}")
+
+
+def validate_summary(summary: Any, systems: list[dict[str, Any]]) -> None:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `summary`

(ANN401)